ref(metrics): Pass org_id to indexer.resolve

jjbayer opened the pull request:
* Add `org_id` to signature of helper functions calling `indexer.resolve`.
* Add `org_id` to all calls to those helper functions.

See also https://github.com/getsentry/sentry/pull/32821 and https://github.com/getsentry/sentry/pull/32825.

MeredithAnya committed:
ref(metrics-indexer): Change indexer resolve signature

jjbayer committed:
ref(metrics): Pass org_id to indexer.resolve

jjbayer commented:
@ahmedetefy I'm also working on a change that adds the appropriate org_id to the metrics API, but that turns out to be trickier than I expected, so I will create a follow-up PR for that.

ahmedetefy commented:
@jjbayer does this mean merging this before changing metrics API will break the metrics API?

jjbayer commented:
> @jjbayer does this mean merging this before changing metrics API will break the metrics API?

@ahmedetefy no, we can merge that one into [metrics-indexer/3-21-22-2](https://github.com/getsentry/sentry/tree/metrics-indexer/3-21-22-2) independently.

jjbayer committed:
fix(metrics): Add org_id to calls to resolve in metrics API (#32833)

As in #32830, add the org_id to all calls of indexer.resolve and related
functions in the metrics API code.

jjbayer committed:
test: resolve calls in test_tasks

jjbayer committed:
fix: Revert breaking change and fix some more tests

jjbayer committed:
Merge branch 'metrics-indexer/3-21-22-2' into fix/add-org-id-to-indexer-resolve
